improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -378,25 +378,32 @@ def create_wet_bulb_temperature_cube(
             data=wbt_data)
         return wbt
 
-    def process(self, temperature, relative_humidity, pressure):
+    def process(self, cubes):
         """
         Call the calculate_wet_bulb_temperature function to calculate wet bulb
         temperatures. This process function splits input cubes over vertical
         levels to mitigate memory issues when trying to operate on multi-level
         data.
 
         Args:
-            temperature (iris.cube.Cube):
-                Cube of air temperatures.
-            relative_humidity (iris.cube.Cube):
-                Cube of relative humidities.
-            pressure (iris.cube.Cube):
-                Cube of air pressures.
+            cubes (iris.cube.CubeList or list or iris.cube.Cube):
+                containing:
+                    temperature (iris.cube.Cube):
+                        Cube of air temperatures.
+                    relative_humidity (iris.cube.Cube):
+                        Cube of relative humidities.
+                    pressure (iris.cube.Cube):
+                        Cube of air pressures.
 
         Returns:
             iris.cube.Cube:
                 Cube of wet bulb temperature (K).
         """
+
+        # pylint: disable=unbalanced-tuple-unpacking
+        temperature, relative_humidity, pressure = self._extract_cubes(
+            cubes, ["air_temperature", "relative_humidity", "air_pressure"])
```

Comment:
Can we avoid imbedding iris functions, all we are doing is reraising an iris ConstraintMismatchError, which is probably more descriptive than a TypeError?
```
names = ("air_temperature", "relative_humidity", "air_pressure")
tuple(CubeList(f).extract_strict(n) for n in names)
```